### PR TITLE
Change the title of the "Set the region" and "Plot lines" tutorials

### LINKED ISSUE #1134: Change the title of the "Set the region" and "Plot lines" tutorials
The PyGMT documentation has several tutorials (see the User Guide in the sidebar on https://www.pygmt.org/dev/). Most of the tutorials have titles like "Plotting XXX", "Adding XXX", or "Creating XXX", except for two tutorials.

- The title of the [lines.py](https://github.com/GenericMappingTools/pygmt/blob/master/examples/tutorials/lines.py) tutorial is "Plot lines", and should be changed to "Plotting lines".
- The title of the [regions.py](https://github.com/GenericMappingTools/pygmt/blob/master/examples/tutorials/regions.py) tutorial is "Set the region", and should be change to "Setting the region".

### PR DESCRIPTION
**Pull request by xdshivani**
/format
/test-gmt-dev

**Description of proposed changes**
* The title of the [lines.py](https://github.com/GenericMappingTools/pygmt/blob/master/examples/tutorials/lines.py) tutorial is "Plot lines", and should be changed to "Plotting lines".
* The title of the [regions.py](https://github.com/GenericMappingTools/pygmt/blob/master/examples/tutorials/regions.py) tutorial is "Set the region", and should be change to "Setting the region".

Fixes #1134 

**Reminders**

- [x] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Review by iamayushdas — APPROVED** [review]

**Review by iamayushdas — APPROVED** [review]

File: examples/tutorials/lines.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """
-Plot lines
+Plotting. lines
 ==========
```
Comment (seisman) [inline examples/tutorials/lines.py:3]: I forgot to mention that we should all keep the `=` same length as the title:
```suggestion
Plotting lines
==============
```

File: examples/tutorials/regions.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 """
-Set the region
+Setting the region
 ==============
```
Comment (seisman) [inline examples/tutorials/regions.py:3]:
```suggestion
Setting the region
==================
```

File: examples/tutorials/lines.py [hunk 3]
```diff
@@ -18,7 +18,7 @@
 import pygmt
 
 ########################################################################################
-# Plot lines
+# Plotting lines
 # ----------
```
Comment (seisman) [inline examples/tutorials/lines.py:22]: I revert this change so this PR only focuses on the tutorial titles.
```suggestion
# Plot lines
# ----------
```
